pandas/tests/arrays/categorical/test_constructors.py

```diff
@@ -42,6 +42,12 @@ def test_constructor_empty(self):
         expected = pd.Int64Index([1, 2, 3])
         tm.assert_index_equal(c.categories, expected)
 
+    def test_constructor_empty_boolean(self):
+        # see gh-22702
+        cat = pd.CategoricalIndex([], categories=[True, False])
```

Comment:
Should this be `Categorical` instead of `CategoricalIndex`?  If you want to test `CategoricalIndex`, I think the test should go in `tests/indexes/test_category.py`.

Reply:
Added a test into `tests/indexes/test_category.py` and also changed this over to use `pd.Category`, since the bug is present in both the index and the Category itself. Sharp eye @jschendel!